src/msp_displayport.c

```diff
@@ -26,6 +26,7 @@ uint8_t fc_lock = 0;
 uint8_t disp_mode; // DISPLAY_OSD | DISPLAY_CMS;
 uint8_t osd_ready;
 
+uint8_t vtx_variant[4] = {'H', 'D', 'Z', '0'};
```

Comment:
Not sure how useful this one is... as it really comes down to other vendors also support this new message type which to be honest I don't foresee happening.  Let me know If you want this removed.... as for the others they are really useful.

Reply:
Replaced with VTX_NAME since its already used in platform.